pvlib/solarposition.py

```diff
@@ -1476,3 +1432,12 @@ def sun_rise_set_transit_geometric(times, latitude, longitude, declination,
     sunset = _local_times_from_hours_since_midnight(times, sunset_hour)
     transit = _local_times_from_hours_since_midnight(times, transit_hour)
     return sunrise, sunset, transit
+
+
+def _warn_how_numba(how):
+    # https://github.com/pvlib/pvlib-python/issues/1060
+    # https://github.com/pvlib/pvlib-python/pull/1098
+    if how is not None:
+        warnings.warn("The 'how' parameter is deprecated; numba is now used "
+                      "automatically if it is available.",
```

Comment:
```suggestion
        warnings.warn("The 'how' parameter is deprecated; numba is now used "
                      "automatically if it is available, otherwise numpy is used.",
```